lib/plugins/awsRemoveResources/lib/removeStack.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
+
+const BbPromise = require('bluebird');
+const async = require('async');
+const find = require('lodash').find;
+
+module.exports = {
+  remove() {
+    this.serverless.cli.log('Removing Stack...');
+
+    const stackName = `${this.serverless.service.service}-${this.options.stage}`;
+
+    const params = {
+      StackName: stackName,
+    };
+
+    return this.CloudFormation.deleteStackPromised(params).then(() => {
+      return BbPromise.resolve(params);
```

Comment:
I'm basically faking the `cfData` here as I pass the params from above to the monitor method (I need the `StackName` in monitor to check the status of the removed stacks.